packages/enzyme-adapter-utils/src/Utils.js

```diff
@@ -262,23 +262,40 @@ export function getComponentStack(
 export function simulateError(
   error,
   catchingInstance,
+  catchingType,
```

Comment:
similarly, this makes the commit a breaking change - it'll have to be added at the end.